scripts/install_minikube.sh

```diff
@@ -9,9 +9,8 @@ function install_minikube() {
 
     if ! [ -x "$(command -v minikube)" ]; then
         echo "Installing minikube..."
-        curl --retry 3 -Lo minikube https://storage.googleapis.com/minikube/releases/v1.8.2/minikube-linux-amd64
-        chmod +x minikube
-        ${SUDO} cp minikube /usr/bin/
+        arkade get minikube --version=v1.18.1
+        ${SUDO} mv ${HOME}/.arkade/bin/minikube /usr/local/bin/
```

Comment:
can't it install directly to ``/usr/local/bin``?
or can we make it cleaner in some other way?

Reply:
```bash
➜  ~ arkade get minikube
Downloading minikube
https://github.com/kubernetes/minikube/releases/download/v1.18.1/minikube-linux-amd64
57.02 MiB / 57.02 MiB [------------------------------------------------] 100.00%
Tool written to: /home/yuvalg/.arkade/bin/minikube

# Add (minikube) to your PATH variable
export PATH=$PATH:$HOME/.arkade/bin/

# Test the binary:
/home/yuvalg/.arkade/bin/minikube

# Or install with:
sudo mv /home/yuvalg/.arkade/bin/minikube /usr/local/bin/

```

Strange that they do not support it with a flag